model_api/cpp/models/src/detection_model_ssd.cpp

```diff
@@ -35,6 +35,26 @@ struct InputData;
 
 std::string ModelSSD::ModelType = "ssd";
 
+ModelSSD::ModelSSD(std::shared_ptr<ov::Model>& model, const ov::AnyMap& configuration)
+    : DetectionModel(model, configuration) {
+    auto object_size_iter = configuration.find("object_size");
+    if (object_size_iter == configuration.end()) {
+        if (model->has_rt_info("model_info", "object_size")) {
+            objectSize = stoull(model->get_rt_info<std::string>("model_info", "object_size"));
```

Comment:
Do we really have to provide the ability to configure these parameters because they are tightly connected to the model architecture and derived from it?  My take is that we should not.

Reply:
The idea is that SSD requires these parameters to do postprocessing. In case of adapter SSD does not have access to the model to initialize them, so it fetches them from the configuration provided by the adapter. Everything is transparent to the user. I would like to highlight that these two parameters are not "configurable" so to say they come from the model itself.

Reply:
Ok. I will revert this